app/celery/broadcast_message_tasks.py

```diff
@@ -16,13 +16,15 @@ def send_broadcast_event(broadcast_event_id):
         current_app.logger.info(f'CBC Proxy disabled, not sending broadcast_event {broadcast_event_id}')
         return
 
+    broadcast_event = dao_get_broadcast_event_by_id(broadcast_event_id)
     for provider in current_app.config['ENABLED_CBCS']:
-        # TODO: Decide whether to send to each provider based on platform admin, service level settings, broadcast
-        # level settings, etc.
-        send_broadcast_provider_message.apply_async(
-            kwargs={'broadcast_event_id': broadcast_event_id, 'provider': provider},
-            queue=QueueNames.NOTIFY
-        )
+        if broadcast_event.service.allowed_broadcast_provider in {None, provider}:
```

Comment:
Optional, wondering if so we avoid this same logic potentially being seen across several places in our code in the future we, maybe in time, introduce a `service.enabled_broadcast_providers` or equivalent such that

```
def enabled_broadcast_providers(self):
    if self.allowed_broadcast_provider:
        return [self.allowed_broadcast_provider]
    else:
        return current_app.config['ENABLED_CBCS']
```

note, above could be written pythonically in a one liner

Reasons for why it might be a good thing to move to in the future
- might reduce some code duplication and moves the complexity into one method
- codes in a quite defensive way, I don't want us to risk accidentally having a bug in our code and sending to fewer providers than we should